### ISSUE TITLE
Take over MC Utilization dashboard

### ISSUE THREAD

**Issue by gawertm** [giantswarm/roadmap#2766]
https://giantswarm.grafana.net/d/pszZmQ0Vk/management-cluster-utilization-request-vs-actual-usage?orgId=1

from https://github.com/giantswarm/giantswarm/issues/19894

I also realized some Installation are missing. where is the list in the dashboard coming from?

**Comment by mogottsch** [comment]
I've added the our team label to the dashboard.

Concerning missing installations, I've identified the following:
- gauss
- cuckoo
- giraffe
- glippy
- gmc
- goat
- goose
- gopher
- goten
- icecream
- leopard

I traced the metrics back to the prometheus instances on the respective installations and could not find a reason why they are missing. I need to talk to someone with more knowledge to figure this out.

**Comment by mogottsch** [comment]
I talked to Antonia and we think the issue is not with our dashboard, but with the metrics themselves. 
I will talk to Atlas about this.

**Comment by mogottsch** [comment]
Atlas identified the issues with the missing installations. Read the [slack thread](https://gigantic.slack.com/archives/C01176DKNP4/p1694599058630499) for more information.

Tldr: The reason for missing metrics varies across installations, but they have an issue tracking the problem of the customer ones (leopard & cuckoo).
